README.md

```diff
@@ -290,6 +302,19 @@ Manual DNS configuration can be specified via a list of addresses
  properties, respectively. If specified, it determines the route metric
 for DHCP assigned routes and the default route, and thus the priority for multiple interfaces.
 
+Static route configuration can be specified via a list of routes given in the `route`
+option. The default value is an empty list. Each route is a dictionary with the following
+entries: `network`, `prefix`, `gateway` and `metric`. `network` and `prefix` together specify
+the destination network. CIDR notation or network mask notation are not supported yet. If the
+boolean option `route_append_only` is `True`, the specified routes are appended to the
+existing routes, if it is `False` (default), the current routes are replaced. Setting this
+option without setting `route` has the effect of preserving the current static routes. The
```

Comment:
"Setting this option without...." this sentence is not clear to me. In fact, if you do not specify "route" nor "route_append_only", the default is
```
    - route
    - route_append_only: false
```
in this case, it will configure no routes (or remove all existing routes on update).

Maybe it should say: "Setting this option to `true` without setting `route` has the effect..."

Reply:
That was the intent, by "setting the option" I meant "to true", setting it to False would be actually unsetting...

Reply:
"setting the option" means to me just specifying it (instead of omitting it). Can we say "setting the option to `true`"? That's all I really had to say, in summary.

Reply:
I made the change that you suggest for clarity.